pkg/git/libgit2/managed/ssh.go

```diff
@@ -89,6 +102,9 @@ type sshSmartSubtransport struct {
 	currentStream *sshSmartSubtransportStream
 }
 
+var aMux sync.RWMutex
+var sshClients map[string]*ssh.Client = make(map[string]*ssh.Client)
```

Comment:
Some description about this collection of ssh clients would be nice in order to easily understand the expected key.